lib/view/page/create_idea_page.dart

```diff
@@ -0,0 +1,107 @@
+import 'package:flutter/material.dart';
+import 'package:idea_tracker/view/widget/state_management/base_view.dart';
+import 'package:idea_tracker/controller/page/create_idea_controller.dart';
+
+class CreateIdeaPage extends StatelessWidget {
+  final String title;
+  final _formKey = GlobalKey<FormState>();
+  final _focusNode = FocusNode();
+  void _onFieldSubmitted(String value) {
+    _focusNode.requestFocus();
+  }
+
+  CreateIdeaPage({this.title});
+
+  @override
+  Widget build(BuildContext context) {
+    return BaseView<CreateIdeaController>(
+      builder: (context, controller, child) {
+        return Scaffold(
+          appBar: AppBar(
+            centerTitle: true,
+            title: Text(title),
+            leading: IconButton(
+              icon: Icon(Icons.arrow_back),
+              onPressed: () {
+                Navigator.of(context).pop();
+              },
+            ),
+          ),
+          body: Builder(
+            builder: (context) {
+              // Temporary solution to show some feedback
+              if (controller.ideaCreatedSuccesfully) {
+                WidgetsBinding.instance.addPostFrameCallback((_) {
+                  Scaffold.of(context).showSnackBar(SnackBar(
+                    content: Text("Your idea has been created"),
+                  ));
+                });
+              }
```

Comment:
Use a callback function on the controller (registered in the onControllerReady function) instead of checking a boolean value. Comment if you need more instructions here.

Reply:
Do you mean passing a callback to the controller in onCotrollerReady to calling `showSnackBar` ?

BTW, the current code in lib/view/widget/state_management/base_view.dart specifies `onModelReady()` instead of `onCotrollerReady`